config/_default/menus/main.en.yaml

```diff
@@ -2235,16 +2235,21 @@ menu:
       parent: error_tracking_backend
       identifier: error_tracking_be_logs
       weight: 104
+    - name: Sentry SDK
```

Comment:
Perhaps we add a new section for Guides and this is the first guide cc @iadjivon